.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+Ensure the title of this pull-request starts with the component name followed by a colon, e.g.
+> ncm-example: demonstrate what titles should look like
+
+Describe the change you are making here, in particular we would like to know:
+
+* Why the change is necessary.
+* What backwards incompatibility it may introduce.
```

Comment:
Just got a colleague to check, only us developers are privileged enough to set the labels.